pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -136,7 +136,10 @@ type BrokerConfig struct {
 	// and the operator supposes that the user is aware of how scheduling is done by kubernetes
 	// Affinity could be set through brokerConfigGroups definitions and can be set for individual brokers as well
 	// where letter setting will override the group setting
-	Affinity *corev1.Affinity `json:"affinity,omitempty" protobuf:"bytes,18,opt,name=affinity"`
+	Affinity           *corev1.Affinity           `json:"affinity,omitempty" protobuf:"bytes,18,opt,name=affinity"`
```

Comment:
The  `protobuf:"bytes,18` seems to be a leftover from a previous PR. I'd remove it as it's not used.